### PR TITLE
247 giving min amount

### LINKED ISSUE #247: Giving minimum amount
- [ ] ContributionForm should only allow giving amounts over $1.00 (unless bank account transactions cost more than that)

### PR DESCRIPTION
**Pull request by IsaacHardy**
Fixes #247 

## Creator

- [ ] Build relevant tests if any apply
- [x] Ensure there are no new warnings
- [x] Upload an animated GIF showcasing the solution (if it applies)
- [x] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [x] Review new test logic

### CODE REVIEW

File: src/@ui/forms/ContributionForm/index.js [hunk 1]
```diff
@@ -259,7 +259,7 @@ export class ContributionFormWithoutData extends Component {
           {this.props.isLoggedIn ? (
             <Button
               onPress={this.props.handleSubmit}
-              disabled={!(this.totalContribution > 0) || !this.props.isValid}
+              disabled={!(this.totalContribution >= 1.0) || !this.props.isValid}
```
Comment (burkeshartsis) [inline src/@ui/forms/ContributionForm/index.js]: so `isValid` should really be handling this entirely. I recommend updating the YUP validation schema for the contribution values [here](https://github.com/NewSpring/Apollos/blob/247-give-min/src/%40ui/forms/ContributionForm/index.js#L325) to use [`number.min(1)`](https://github.com/jquense/yup#numberminlimit-number--ref-message-string-schema)

**Comment by IsaacHardy** [comment]
@burkeshartsis Updated Yup validation models.

**Comment by burkeshartsis** [comment]
I'll finish up my review and get this merged today.

**Comment by IsaacHardy** [comment]
@burkeshartsis Each giving amount is a separate transaction where no transaction can be less than $1. We actually pay more money in fees than we receive when the amount is less than a $1 so having this type of behavior is acceptable.

**Pull request merged**